packages/react/src/auto/index.tsx

```diff
@@ -0,0 +1,5 @@
+import type { AutoFormProps } from "./AutoForm.js";
+
+export interface AutoAdapter<T = any, F = any> {
+  AutoForm: React.ComponentType<AutoFormProps<any, any, any, any> & F>;
```

Comment:
Doesn't this need all the other components in it too?

Reply:
I'm not sure I actually follow what this interface does other than abstract between the polaris and mui autoform for tests, in which case we only need it for the autoform at the moment.